pages/index.js

```diff
@@ -63,7 +63,7 @@ const NewProducts = ({ products }) => {
 								// @TODO Need to add support for Group product.
 								( undefined !== item && 'GroupProduct' !== item.__typename) ?  (
 								<div className="product-container col-md-3 mb-5" key={item.id}>
-									<Link as={`/product/${item.slug}-${item.productId}`} href={`/product?slug=${item.slug}-${item.productId}`}>
+									<Link as={`/product/${item.slug}-${item.databaseId}`} href={`/product?slug=${item.slug}-${item.databaseId}`}>
```

Comment:
Lets add a @TODO.
This should be just a slug of the product and we don't need database it.

Reply:
The `databaseId`  is used here to fetch the single product though: https://github.com/rtCamp/wp-decoupled/blob/master/pages/product.js#L38

Reply:
@m-muhsin 
For now its ok. Howevever, we should add a TODO to change it later, to fetch the product by slug.

Reply:
Sure @imranhsayed - thanks for the suggestion!